examples/WarmCopper/include/simulation_defines/param/particleConfig.param

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2013-2017 Axel Huebl, Rene Widera, Benjamin Worpitz,
+ *                     Richard Pausch
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#pragma once
+
+#include "particles/startPosition/functors.def"
+#include "particles/manipulators/manipulators.def"
+#include "nvidia/functors/Add.hpp"
+#include "nvidia/functors/Assign.hpp"
+#include "particles/traits/GetAtomicNumbers.hpp"
+
+namespace picongpu
+{
+
+namespace particles
+{
+
+    /** a particle with a weighting below MIN_WEIGHTING will not
+     *      be created / will be deleted
+     *  unit: none */
+    constexpr float_X MIN_WEIGHTING = 10.0;
+
+    constexpr uint32_t TYPICAL_PARTICLES_PER_CELL = 2;
+
+namespace manipulators
+{
+
+    CONST_VECTOR(float_X, 3, DriftParam_direction, 1.0, 0.0, 0.0);
+    struct DriftParam
+    {
+        static constexpr float_64 gamma = 1.39139;
```

Comment:
this is directly documented in the line below where this struct is used via `Assign200keVDrift`, but I can add it...

```C++
// ...
        static constexpr float_64 gamma = 1.39139;
        const DriftParam_direction_t direction;
    };
    /* definition of SetDrift start*/
    using Assign200keVDrift = DriftImpl< DriftParam, nvidia::functors::Assign >;
```